steve/util.py

```diff
@@ -216,18 +391,21 @@ def get_project_config_file_name():
     return SteveConfig.file_name
 
 
-def create_project_config_file(dir_path):
+def create_project_config_file(dir_path, cfg=None):
```

Comment:
Just that I am more used to projects where it is not good style to have defaults other than None.